multimodal/src/autogluon/multimodal/models/timm_image.py

```diff
@@ -48,13 +50,32 @@ def __init__(
         super().__init__()
         # In TIMM, if num_classes==0, then create_model would automatically set self.model.head = nn.Identity()
         logger.debug(f"initializing {checkpoint_name}")
-        self.checkpoint_name = checkpoint_name
+        if os.path.exists(checkpoint_name):
+            checkpoint_path = f"{checkpoint_name}/pytorch_model.bin"
+            try:
+                with open(f"{checkpoint_name}/config.json") as f:
+                    self.config = json.load(f)
```

Comment:
This `self.config` will be overwritten later in line 76?

Reply:
Good catch, I believe the `self.config` on line 76 should be moved.